server_admin/topics/realms/ssl.adoc

```diff
@@ -8,21 +8,45 @@ Browsers and applications that interact with the realm honor the SSL/HTTPS requi
 WARNING:  {project_name} generates a self-signed certificate the first time it runs.  Please note that self-signed certificates are not secure, and should only be used for testing purposes.  It is highly recommended that you install a CA-signed certificate on the {project_name} server itself or on a reverse proxy in front of the {project_name} server.  See the link:{installguide_link}[{installguide_name}].
 
 .Procedure 
-. Click *Realm Settings* in the menu.
 
+ifeval::[{project_product}==true]
+. Click *Realm Settings* in the menu.
 . Click the *Login* tab.
-+
```

Comment:
The formatting is wrong when I build the SSO version.
Add back the line with the + (plus sign).